### ISSUE TITLE
The PIMD protocol supports hello-interval less than 30s? Does the pimd protocol support Vxworks? 

### ISSUE THREAD

**Issue by zcauto** [troglobit/pimd#214]
Hi,
I have two questions,
1. From the pimd.conf file, the hello interval of the pimd protocol is not less than 30s. Will there be any problems if I set the hello interval to 5s?
2. Can the pimd protocol run normally under Vxworks?

**Comment by troglobit** [comment]
Why have you opened a new issue with the same questions?  Isn't this a duplicate of #213 which I replied to this morning?

**Comment by zcauto** [comment]
Oh, I'm very sorry. I guess it may be that when I submitted #213, the network condition was not good, and I didn’t see it in the list after I submitted it. 
Therefore, if the CPU load allows and I reduce all the timer intervals in the PIMD protocol, the protocol still cannot work? I have an application scenario now, and I have to change the timer interval to be smaller. Do you have any good suggestions?

**Comment by troglobit** [comment]
The documentation is clear, the implementation has limits. You need to look over your application scenario. That or read up on the protocol, there is to my knowledge no application specific need for routers to send PIM Hello messages more often than the RFC details.

Also, none of the issues you've filed are issues with pimd, but instead questions. Again, I implore you to use the Discussions tab for this.

Please do not open more issues with the project again for support in your work, at least not until you have actual findings in the project to report.
